contracts/utils/Packing.sol

```diff
@@ -0,0 +1,40 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.20;
+
+/**
+ * @dev Helper library packing and unpacking multiple values into bytes32
+ */
+library Packing {
+    type Uint128x2 is bytes32;
```

Comment:
For now we only have `(uint128, uint128) <> bytes32`, but we can imagine having other types, such as `uint64x4`

Everything here is designed to scale to oither types.